src/deploy/ecs.py

```diff
@@ -97,3 +98,25 @@ def _has_matching_tags(service):
             return None
         else:
             return matched_services[0]
+
+    def list_service_tasks(self, service):
+        """
+        Given a service (e.g. bag-unpacker) and an environment (e.g. prod),
+        return a list of tasks running within that service
```

Comment:
The docstring says this method takes an environment, but I only see a service in the argument list.

Reply:
It only takes a service

Reply:
Then the docstring should be updated.